apps/core/scenes/results/Results.js

```diff
@@ -76,7 +76,13 @@ class Results extends React.Component<Props> {
     props: ReturnResultsQueryResponse | OneWayResultsQueryResponse,
   ) => {
     const { searchData } = props;
-    return <ResultsList data={searchData} onBookPress={this.handleBookPress} />;
+    return this.props.layout < LAYOUT.largeMobile ? (
+      <View>
```

Comment:
It works but after some testing I'm slightly worried about performance.
Because by this we are displaying all results at once (instead of adding them during scroll), which can be a quite huge number. And by this website can become really slow for example during resize events.
So we will probably need to find some way how to keep current behaviour when cards are created during scroll but with whole page scrollable.